Unshaky/nl.lproj/MainMenu.strings

```diff
@@ -0,0 +1,25 @@
+/*
+  MainMenu.strings
+  Unshaky
+
+  Created by Siebrand Mazeland on 2019-04-24.
+  CC BY 4.0 Siebrand Mazeland
+*/
+
+/* Class = "NSMenuItem"; title = "Open debug window"; ObjectID = "Iv3-MX-v73"; */
+"Iv3-MX-v73.title" = "Debugvenster openen";
+
+/* Class = "NSMenuItem"; title = "-"; ObjectID = "T9Q-qI-cKF"; */
+"T9Q-qI-cKF.title" = "-";
+
+/* Class = "NSMenuItem"; title = "Configuration"; ObjectID = "TGg-Zy-iYa"; */
+"TGg-Zy-iYa.title" = "Instellingen";
+
+/* Class = "NSMenuItem"; title = "Check for Updates..."; ObjectID = "f5l-nJ-ecO"; */
+"f5l-nJ-ecO.title" = "Controleren op updates...";
```

Comment:
It's a button to check for updates. It's not check*ing* for updates yet.

```suggestion
"f5l-nJ-ecO.title" = "Controleer op updates...";
```

Reply:
I disagree with this one.